### ISSUE TITLE
NuGet cmdLine 3.6 beta does not honor -Prop Configuration = Release

### ISSUE THREAD

**Issue by TheBigRic** [NuGet/Home#3432]
I used NuGet 3.3 before for creating packages using the commandline with this command:

```
nuget pack %project_file% -Build -Symbols 
   -IncludeReferencedProjects -Properties Configuration=Release
```

Because of this issue #1922, I tried to use 3.5 RC1, but this gave me still some problems, so I decided to try 3.6. This builded the project in Debug and then tried to package (zip) the files from the \bin\release folder, which was empty. Sometimes it failed, Other times it succeeded, however looking in the created package, it packed only some default xml files and nuspec file.

I got this working by removing `-Properties Configuration=Release` from the commandline and change my .csproj to let the default build config point to Release:

``` xml
<Configuration Condition=" '$(Configuration)' == '' ">Release</Configuration>
```

**Comment by rrelyea** [comment]
please investigate

**Comment by rohit21agrawal** [comment]
@TheBigRic could you share more details of the kind of project it was? or even better, could you attach a small repro project so I am able to repro this issue on my end?

**Comment by TheBigRic** [comment]
[Nuget.Repro.zip](https://github.com/NuGet/Home/files/469782/Nuget.Repro.zip)

I attached a zip file which contains 2 solutions:
1. Default build config = debug
2. Default build config = release

Solution 1 has this config in de csproj files:

``` xml
<Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Solution 2 has this config in de csproj files:

``` xml
<Configuration Condition=" '$(Configuration)' == '' ">Release</Configuration>
```

Both solutions contain 2 batch files for creating the package. One fails, the other succeeds.
The only differ in this line:

```
%nuget% pack %project_file% -Build -IncludeReferencedProjects -Prop Configuration=Release
%nuget% pack %project_file% -Build -IncludeReferencedProjects
```

I created 2 solutions to proof the problem exists in both release and debug environments
